eks/external-secrets/fleet.yaml

```diff
@@ -5,3 +5,27 @@ helm:
   chart: external-secrets
   takeOwnership: true
   releaseName: external-secrets
+  values:
+    resources:
+      requests:
+        cpu: 2m       # 0.001 * 1500 = 1.5m ≈ 2m
+        memory: 52Mi  # 47 * 1.1 = 51.7Mi ≈ 52Mi
+      limits:
+        cpu: 2m       # 0.001 * 2000 = 2m
```

Comment:
Setting limits (more below) to such low values seems wrong. Should we allow for bursts and set this a bit higher?